### PR TITLE
[stable/vpa] Created capability to add annotations to vpa deployments and fixed pdb labels

### LINKED ISSUE #1293: [stable/vpa] Expose ability to add deployment annotations in the vertical pod autoscaler helm chart
**Is your feature request related to a problem? Please describe.**
Cannot add annotations to the deployments of the helm chart for the vertical pod autoscaler. Hinders our ability to use projects like the [reloader](https://github.com/stakater/Reloader) which in our case we need in order to restart the VPA pods to pick up new webhook certs managed by cert-manager.

**Describe the solution you'd like**
Provide the ability to add annotations to the deployments similar to the other resources.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by murali-krishna**
**Why This PR?**
Created capability to add annotations to vpa deployments and fixed pdb labels

Fixes #1293 #1294 

**Changes**
Changes proposed in this pull request:

* Created capability to add annotations to vpa deployments
* Set `app.kubernetes.io/name` pdb label to match the value in helper file

**Checklist:**

* [X] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [X] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/vpa/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: vpa
 description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 type: application
-version: 2.4.0
+version: 2.4.1
```
Comment (sudermanjr) [inline stable/vpa/Chart.yaml]: Since this adds new functionality, it should be considered a minor change, not a patch. 

```suggestion
version: 2.5.0
```

**Review by sudermanjr — APPROVED** [review]

**Comment by sudermanjr** [comment]
Thanks!

**Comment by bizrad** [comment]
Hmm, this is breaking for me using like this:
```
annotations:
  "foo.io/deploy-repo": "https://gitlab.com/foo/myrepo"
  "foo.io/slack-channel": "myteam"
```
I'm getting this error
```
        Helm install failed: YAML parse error on
        vpa/templates/recommender-deployment.yaml: error converting YAML to
        JSON: yaml: line 5: did not find expected key
```

**Comment by murali-krishna** [comment]
> Hmm, this is breaking for me using like this:
> 
> ```
> annotations:
>   "foo.io/deploy-repo": "https://gitlab.com/foo/myrepo"
>   "foo.io/slack-channel": "myteam"
> ```
> 
> I'm getting this error
> 
> ```
>         Helm install failed: YAML parse error on
>         vpa/templates/recommender-deployment.yaml: error converting YAML to
>         JSON: yaml: line 5: did not find expected key
> ```

@bizrad Thanks for reporting this. I have fixed it [here](https://github.com/FairwindsOps/charts/pull/1307)

**Pull request merged**